src/main.js

```diff
@@ -509,7 +580,7 @@ class MiniGraphCard extends LitElement {
   renderSvg() {
     const { height } = this.config;
     return svg`
-      <svg width='100%' height=${height !== 0 ? '100%' : 0} viewBox='0 0 500 ${height}'
+      <svg width='100%' height=${height !== 0 || this.sections ? '100%' : 0} viewBox='0 0 ${this.graphWidth} ${this.graphHeight}'
```

Comment:
Looking at this... Why would we ever set height to zero? If this is some kind of broken remnant, we could as well remove it. It might make our life easier with min and max height in the styles. But I do not fully see the implications yet.

Reply:
It seems this is the old way of the hidding graph part.

I don't see a reason to keep this since we have a option to hide it through `show.graph`.

I kept it there because I did not want to modify cards behaviour on dashboards other than `sections`.

Reply:
OK. Let's stick to adding sections. We can still remove this in a seperate effort.